docs/glsl-fuzz-develop.md

```diff
@@ -15,13 +15,6 @@ page](glsl-fuzz-releases.md).
 * [Maven](https://maven.apache.org/)
 * [Python 3.5+](https://www.python.org/)
 
-* For our Android workers: [Android SDK & NDK](android-notes.md)
```

Comment:
I would keep all this. The SDK is still needed for our Android GLES worker, and the SDK+NDK is needed to build amber (which is technically part of our Android Vulkan worker). You could change "For the Vulkan desktop worker", but actually I think it is fine to leave this as-is. We could tweak the "Out workers are..." I suppose; maybe: "Our *workers* are applications that run on (or near) the device you wish to test;...".

Reply:
Reverted.